src/gms/generate_branch_list.py

```diff
@@ -17,13 +17,18 @@ def Generate_branch_list(stream_network_dissolved, branch_id_attribute, output_b
     # write
     stream_network_dissolved.to_csv(output_branch_list,sep= " ",index=False,header=False)
 
+    # Add branch zero ID to branch list
+    if branch_zero:
+        with open(output_branch_list,'a') as branch_lst:
+            branch_lst.write(f'{branch_zero}')
 
 if __name__ == '__main__':
 
     parser = argparse.ArgumentParser(description='Create branch list')
     parser.add_argument('-d','--stream-network-dissolved', help='Dissolved stream network', required=True)
     parser.add_argument('-b','--branch-id-attribute', help='Branch ID attribute to use in dissolved stream network', required=True)
     parser.add_argument('-o','--output-branch-list', help='Output branch list', required=True)
+    parser.add_argument('-z','--branch-zero', help='Branch Zero ID', required=False)
```

Comment:
What type of data are you expecting here: a "0"  or a "yes" or some other value?  You don't have to put in a datatype here as it will assume string.  No biggie. Maybe put a sample usage in this "__main__" block. Not all permutations just an idea of roughly what a person might see. Especially if we build a unit_test against it someday.